tests/unit/bokeh/io/test_state.py

```diff
@@ -44,7 +44,7 @@ def test_creation(self) -> None:
     def test_default_file_resources(self) -> None:
         s = bis.State()
         s.output_file("foo.html")
-        assert s.file.resources.minified, True
+        assert s.file.resources.minified is True
```

Comment:
```suggestion
        assert s.file.resources.minified
```

Reply:
For this one I these I think the intent is to confirm the actual bool value, not just that it is truth-y